adaptdl/adaptdl/speedup.py

```diff
@@ -171,18 +176,36 @@ def __call__(self, nodes, replicas, return_local_bsz=False):
         self._mem_speedup[mem_indices] = speedup[ret_indices]
         self._mem_local_bsz[mem_indices] = local_bsz[ret_indices]
 
+        ret_local_bsz, ret_acc_steps = self._partition_local_bsz(ret_local_bsz)
+
         if isscalar:
             ret_speedup = ret_speedup.item()
-            ret_local_bsz = ret_local_bsz.item()
-
-        return ((ret_speedup, ret_local_bsz)
-                if return_local_bsz else ret_speedup)
+            ret_local_bsz = int(ret_local_bsz.item())
+            ret_acc_steps = int(ret_acc_steps.item())
+        if return_local_bsz:
+            return (ret_speedup, (ret_local_bsz, ret_acc_steps))
+        else:
+            return ret_speedup
+
+    def _partition_local_bsz(self, local_bsz):
```

Comment:
I think some variable renaming is in order. `local_bsz` is exposed through the public API and means the size of the physical batches produced by the dataloader. We should stick with that meaning throughout our code too. Similarly, max_local_bsz should mean the upper-bound of physical batches produced by the dataloader.

We'll need to pick a different name to represent `local_bsz * acc_steps`. Maybe `accum_bsz` or `acc_local_bsz` or something like that?

Currently `local_bsz` inconsistently refers to different things, which makes the code a bit confusing.